spec/coradoc/parser/asciidoc/content_spec.rb

```diff
@@ -242,6 +275,33 @@
       expect(table[:rows][1][:cols][2][:text]).to eq("john.doe@example.com")
     end
 
+    it "parses the table block 2" do
+      content = <<~DOC
+        .Person table
+        |===
+        | *first_name* | last_name | email
+        | John | Doe | john.doe@example.com
+        | | doe | jennie.doe@example.com
+        |===
+      DOC
+
+      ast = Asciidoc::ContentTester.parse(content)
+      table = ast.first[:table]
+
+
+      obj = {:table=>
```

Comment:
Why is this not a Ruby object tree? There should **NOT** be any hashes.

Reply:
Coradoc does:

Coradoc text -> AST -> Ruby object tree

In particular, those tests are for the first step. And AST is represented with Ruby hashes.